app/components/shared/widget-inputs/WTextAreaInput.vue

```diff
@@ -1,11 +1,18 @@
 <template>
-  <textarea
+  <span>
+    <textarea
       type="text"
       :placeholder="metadata.placeholder"
```

Comment:
Little fix that I forgot about - replace `metadata.placeholder` to `options.placeholder`. The `metadata` property can be undefined while `options` always exist.  After that, it's good for me.

Reply:
@holiber done! thanks for info